bbpower/compsep.py

```diff
@@ -579,6 +579,12 @@ def lnprob(self, par):
         if not np.isfinite(prior):
             return -np.inf
 
+        return prior + lnlike(par)
```

Comment:
I imagine you need `self.lnlike` here

Reply:
If you can run one of the tests to make sure that things haven't broken, that'd be great. We need to set up proper unit tests to check this automatically, but we aren't there yet.

Reply:
If you can run one of the tests to make sure that things haven't broken, that'd be great. We need to set up proper unit tests to check this automatically, but we aren't there yet.

Reply:
Thanks for pointing this out! I will rerun some tests before pushing the next commit.

Reply:
I tested the code on my laptop: `test/run_power_specter_test.sh` (with PolyChord) and `test/run_polychord_test.sh` (with emcee) both run successfully.